package.json

```diff
@@ -7,5 +7,11 @@
     "fclone": "1.0.11",
     "winston-log-and-exit": "1.0.6",
     "yargs": "3.30.0"
+  },
+  "devDependencies": {
+    "@types/node": "10.17.24",
+    "@types/winston": "2.2.0",
+    "@types/yargs": "15.0.5",
+    "winston": "2.2.0"
```

Comment:
hmm weird that we didn't already ahve `winston` as a dependency considering it was used before

Reply:
I'll check on this; it might have been a transitive dependency.